src/PresentationalComponents/Cards/Pathways.js

```diff
@@ -206,7 +207,9 @@ const Resolution = (props) => {
         </GridItem>
         <GridItem span={5}>
           <CardTitle>{intl.formatMessage(messages.reclvl)}</CardTitle>
-          <RecommendationLevel {...props} />
+          <CardBody className={`body`}>
```

Comment:
Just `className="body"` if it's not needed to interpolate/concat something with a(nother) variable.